migration/transformation_schema_2_to_1.yaml

```diff
@@ -0,0 +1,134 @@
+# This file provides specific mappings for metric data between Java 2.x to Java 1.x
+# The format should be the same as the otel file_format for schema files.
+file_format: 1.1.0
+# TODO: Fix URL
+schema_url: https://github.com/signalfx/splunk-otel-java/migration/transformation_schema_2_to_1.yaml
+versions:
+  # Java version 2.0.0 is the new version. The following mappings translate
+  # FROM the new/correct/stable names back to the previous names.
+  # The names to use when applying attribute renaming are the NEW names.
+  # <new_name>: <old_name>
+  # We want to translate TO the old names as a migration step
+  2.0.0:
+    metrics:
+      changes:
+        #--------------------------------------------------------------
+        # Memory metrics
+        - rename_metrics:
+            jvm.memory.used:               runtime.jvm.memory.used           # micrometer
+            jvm.memory.committed:          runtime.jvm.memory.committed      # micrometer
+            jvm.memory.limit:              runtime.jvm.memory.max            # micrometer
+            jvm.memory.used_after_last_gc: runtime.jvm.memory.usage.after.gc # micrometer
+#            jvm.memory.init:               process.runtime.jvm.memory.init  # experimental
+        - rename_attributes:
+            attribute_map:
+              jvm.memory.type: area
+              jvm.memory.pool.name: pool
+            apply_to_metrics:
+              - jvm.memory.used
+              - jvm.memory.committed
+              - jvm.memory.limit
+              - jvm.memory.used_after_last_gc
+#              - jvm.memory.init
+
+        #--------------------------------------------------------------
+        # Threads metrics
+        - rename_metrics:
+            jvm.thread.count: runtime.jvm.threads.live # micrometer
+        - rename_attributes:
+            attribute_map:
+              jvm.thread.daemon: daemon
+            apply_to_metrics:
+              - jvm.threads.count
+
+        #--------------------------------------------------------------
+        # Classes metrics
+        - rename_metrics:
+#            jvm.class.loaded:   process.runtime.jvm.classes.loaded
+            jvm.class.unloaded: runtime.jvm.classes.unloaded  # micrometer
```

Comment:
this one should also have the aggregation temporality mismatch, how would you express that?

Reply:
There is no way in this format that I'm aware of to express a change in temporality. Maybe there is, but I also couldn't find the spec for the format and was just riffing on what exists today across 24 versions.

Reply:
If it's a problem we can always remove it.